cfme/fixtures/networks.py

```diff
@@ -29,18 +29,20 @@ def test_something(targeted_refresh)
                     targeted_refresh.register_target('ref-aaaabb', 'Router named TEST')
     """
     evm_log = '/var/www/miq/vmdb/log/evm.log'
-    needle_template = r'^.*Collection of targets with id.*:ems_ref=>"{}".*$'
-    merkyl_inspector.add_log(evm_log)
-    merkyl_inspector.reset_log(evm_log)
-    targets = set()  # { (ems_ref, comment), (ems_ref, comment), ... }
+    needle_template = r'.*Collection of targets with id.*:ems_ref=>"{}".*'
+    targets = {}
```

Comment:
So for a set we had a comment to document its structure. Please consider doing something similar for dict.